src/data/faq.json

```diff
@@ -3237,11 +3237,21 @@
                     "anchor": "backup",
                     "description": "Backups refer to versions of a project at a certain point in time in the past, which can later be used to restore a project to its previous state."
                 },
+                {
+                    "term": "Basic Immunisation",
+                    "anchor": "basic_immunisation",

```

Comment:
Suggest to use the exact spelling used in the app i.e. Immunization, not Immunisation, also for the anchor. (The app normally uses US not UK spelling.)

Reply:
This conversation is marked as resolved although there are still discrepancies in the spelling between the header (immunisation) and the text (immunization).